<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diem_tongket</class>
 <widget class="QWidget" name="diem_tongket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="BangDiemTongKet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>791</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>BẢNG ĐIỂM TỔNG KẾT</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>811</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_malop">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>811</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lớp: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TraTongKet">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tra Bảng Điểm</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>561</width>
     <height>68</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Mã lớp</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit__MaLop">
      <property name="placeholderText">
       <string>Chữ cái</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>tableView</zorder>
  <zorder>BangDiemTongKet</zorder>
  <zorder>label_5</zorder>
  <zorder>label_malop</zorder>
  <zorder>pushButton_TraTongKet</zorder>
  <zorder>gridLayoutWidget_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
